content/eosio/public-apis/reference/network-endpoints.md

```diff
@@ -14,11 +14,18 @@ The _dfuse_ API is available for multiple EOSIO networks. Should you need it on
 
 Chain ID: `aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906`
 
+mainnet.eos.dfuse.io has been deprecated, there are now two ways to access EOS mainnet using dfuse
+
+1. dfuse Community Edition (hosted by [EOS Nation](https://eosnation.io/))
```

Comment:
I suggest we list our dedicated Enterprise option first and than listing access to EOS Nation endpoint.